### DISCUSSION TITLE
Search into a text field with like operator

### DISCUSSION THREAD

**Discussion by apolopistache** [discoveryjs/jora#38] [Q&A]
I'm looking for a way to search into a string field with a "like" operator or something like that.
Can you give a sample on how to do that or a sample with regular expressions.
I tried the ~ = operator without any success
lines.[name~ ="Mercha"]

Thanks you

**Comment by lahmatiy** [comment]
First of all, no spaces in operator, i.e. `~=`. You can use a regexp on the right side, e.g. `.[name ~= /foo/]`, ~but a string in simple cases should work either~ (the operator doesn't work with a string at the moment for some reasons, will check why later, that's might be a bug).
